handler.cpp

```diff
@@ -64,17 +64,41 @@ RequestHandler::Status StaticHandler::Init(const std::string& uri_prefix, const
             serve_path = config.statements_[i]->tokens_[1];
         }
     }
-
+    this->uri_prefix = uri_prefix;
     return RequestHandler::PASS;
 }
 
 // Attempts to serve file that was requested by the request object
 // If not possible, we return fail signal for outside class to handle
 // calling not found handler
 RequestHandler::Status StaticHandler::HandleRequest(const Request& request, Response* response) {
-    std::string abs_path = serve_path + "/" + request.uri();
-    std::cout << "Attempting to serve file from: " << abs_path << std::endl;
+    
+    std::string abs_path = serve_path;
+    //get the file name/path from the request url
+    std::string file_path = request.uri().substr(uri_prefix.length());
+
+    //Code to add the / between the file path and serve_path properly
```

Comment:
what do you think about moving this bit that preprocesses our serve_path to in the init file? This computation should only need to happen once and after that we use the same serve_path every time right?